### PR TITLE
[R2] Ordered list

### PR DESCRIPTION
**Pull request by g-stamatis**
https://taskman.eionet.europa.eu/issues/147803
https://taskman.eionet.europa.eu/issues/147815

### CODE REVIEW

File: src/ui/List/DefaultList.stories.js [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
 export default {
   title: 'Components/List',
   component: List,
   argTypes: {},
 };
 
-const Template = (args) => <List {...args}></List>;
+const Template = (args) => (
+  <Container>
+    <List {...args}></List>
```
Comment (avoinea) [inline src/ui/List/DefaultList.stories.js:12]: @g-stamatis All list should use HTML semantic elements for listing. Thus, here we should use:

```
<ul>
  <li>
```

File: src/ui/List/DescriptionList.stories.js [hunk 2]
```diff
@@ -48,42 +41,37 @@ export default {
...
-            <List.Header>{item.header}</List.Header>
-            <List.Description floated={floated}>
-              {item.content}
-            </List.Description>
-          </List.Content>
-        </List.Item>
-      ))}
-    </List>
+    <Container>
+      <List animated={animated} divided={divided}>
```
Comment (avoinea) [inline src/ui/List/DescriptionList.stories.js:47]: @g-stamatis All list should use HTML semantic elements for listing. Thus, here we should use:

```
<dl>
  <dt></dt>
   <dd></dd>
</dl>
```

File: src/ui/List/OrderedList.stories.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+        defaultValue: { summary: ' "" ' },
+      },
+    },
+  },
+};
+
+function OrderedList(items, animated, divided, horizontal) {
+  return (
+    <Container>
+      <List as="ul" ordered animated={animated} divided={divided}>
```
Comment (avoinea) [inline src/ui/List/OrderedList.stories.js:36]: @g-stamatis All list should use HTML semantic elements for listing. Thus, here we should use:

```
<ol>
  <li></li>
</ol>
```

**Review by avoinea — CHANGES_REQUESTED** [review]

File: src/ui/List/DefaultList.stories.js (hunk 1 above)
Comment (g-stamatis) [reply src/ui/List/DefaultList.stories.js]: @avoinea To avoid overrides we are using dl/dt in the updated files.

**Review by avoinea — APPROVED** [review]

**Pull request merged**